TestWeb.IntegrationTests/Controllers/ControllerTestsBase.cs

```diff
@@ -80,5 +101,11 @@ protected void VerifyResponse(HttpStatusCode statusCode, bool isSuccess)
             Response.StatusCode.Should().Be(statusCode);
             Response.IsSuccessStatusCode.Should().Be(isSuccess);
         }
+
+        [OneTimeTearDown]
+        public void OneTimeTearDown()
+        {
+            _server.Dispose();
```

Comment:
is there a case for a null check before calling dispose?

Reply:
ah yep, let me add it

Reply:
Cheers!